examples/v1beta1/pipelineruns/pipelinerun-with-final-results.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: sum-and-multiply-pipeline
+spec:
+  params:
+    - name: a
+      type: string
+      default: "1"
+    - name: b
+      type: string
+      default: "1"
+  results:
+    - name: finally-result
+      description: "grabbing results from the finally section"
+      value: $(finally.sum-and-multiply.results.sum)
+
+  tasks:
+    - name: sum-inputs
+      taskRef:
+        name: sum
+      params:
+        - name: a
+          value: "$(params.a)"
+        - name: b
+          value: "$(params.b)"
+    - name: multiply-inputs
+      taskRef:
+        name: multiply
+      params:
+        - name: a
+          value: "$(params.a)"
+        - name: b
+          value: "$(params.b)"
```

Comment:
I don't think we need multiple pipeline tasks here

Reply:
done.